app/decorators/mod_action_decorator.rb

```diff
@@ -325,6 +343,8 @@ def format_description
       "Post rating was #{vals['locked'] ? 'locked' : 'unlocked'} on post ##{vals['post_id']}"
     when "post_unapprove"
       "Unapproved post ##{vals['post_id']}"
+    when "post_owner_reassign"
```

Comment:
This seems entirely unnessesary to add

Reply:
I only did that due to [a few entries existing](https://e621.net/mod_actions?search[action]=post_owner_reassign)
So if anyone sees them in the future, they don't report them as a bug

Reply:
I might as well just delete the existing ones. They contain no actual information anyways

Reply:
I'm not opposed to that, in fact that's probably the better solution with them having no information